### PR TITLE
Set angular build to production

### PR DESCRIPTION
**Pull request by margawron**

### CODE REVIEW

**Comment by margawron** [comment]
https://angular.io/guide/deployment

**Review by latusikl — APPROVED** [review]

File: PhotoPlusFrontend/PhotoPlus/src/environments/environment.prod.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
 export const environment = {
   production: true,
-  hostAddress: 'http://http://35.197.224.101:8090/'
+  hostAddress: 'http://35.197.224.101:8090/'
```
Comment (syphzur) [inline PhotoPlusFrontend/PhotoPlus/src/environments/environment.prod.ts:3]: oh no

**Comment by latusikl** [comment]
In free time please check if the production environment file is loading after running ng build --prod. On remote machine during docker-compose up --build it wasn't.

**Comment by syphzur** [comment]
> In free time please check if the production environment file is loading after running ng build --prod. On remote machine during docker-compose up --build it wasn't.

try using _npm run ng build -- --prod._

**Comment by syphzur** [comment]
try using npm run ng build -- --prod.

**Pull request closed without merging**